Library/Homebrew/cask/audit.rb

```diff
@@ -758,18 +762,18 @@ def audit_livecheck_version
 
       # Respect cask skip conditions (e.g. deprecated, disabled, latest, unversioned)
       skip_info ||= Homebrew::Livecheck::SkipConditions.skip_information(cask)
-      return :skip if skip_info.present?
+      return (@livecheck_result = :skip) if skip_info.present?
```

Comment:
Splitting this into multiple lines would be nice here; the assignment inline is a bit hard to follow.